src/Tools/TransformXMLToJSONUsingXSLT/README.md

```diff
@@ -0,0 +1,72 @@
+# Transformation Tool from XML to Text
+
+Transformation Tool is an application to:.
+
+  - Transform files XML to Text
+  - Create text files
+  - Configure the entry files and set the out put files.
```

Comment:
what's an entry file? Whats an output file? in the context of this tool?

Reply:
A diagram will be added to explain more about the tool and the context of the same.

Reply:
👍